addons/hr/models/hr_employee.py

```diff
@@ -1390,3 +1391,16 @@ def action_new_contract(self):
             'contract_date_start': False,
             'contract_date_end': False,
         })
+
+    def _get_store_avatar_card_fields(self):
+        employee_fields = [
+            "company_id",
+            Store.One("department_id", ["name"]),
+            "work_email",
+            Store.One("work_location_id", ["location_type", "name"]),
+            "work_phone",
+        ]
+        if self.env.user.has_group("hr.group_hr_user"):
```

Comment:
should pass target to `_get_store_avatar_card_fields` and check group of target/user instead